packages/bundles-dynamic/aws.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Is the filename correct? `aws.json`

Reply:
I couldn't find any proper information for this deployment json file usage [here](https://docs.commercetools.com/custom-applications/deployment-examples/aws-s3-cloudfront).
Can you guide me to the docs which helps me understand how this deployment works?

Reply:
Thank you @emmenko.

@nspeet I see there is [doc](https://github.com/commercetools/commercetools-bundles-starter/tree/master/packages/bundles-static#deployment) on deployment for Firebase and AWS, which won't work properly after these PR changes. 
And the links in the document is already broken, it is not found anymore from Commercetools documents. Ex: [Firebase](https://docs.commercetools.com/custom-applications/deployment/example-deployment-firebase)
[AWS](https://docs.commercetools.com/custom-applications/deployment/example-aws-s3-cloudfront)

Do you think we need to setup and provide a proper documentation from our end(Only for Bundles-static and dynamic projects) to perform deployment for both Firebase and AWS. Or this should be handled by users and we will provide support only for the deployment options available in MC custom applications [documentation](https://docs.commercetools.com/custom-applications/deployment-examples)?

Reply:
Hi @praveenkumarct, in the bundles doc let's link to the MC app deployment options latest [doc](https://docs.commercetools.com/custom-applications/deployment-examples)

Reply:
https://github.com/commercetools/commercetools-bundles-starter/pull/231/commits/0588eb2944af98c1e431f43af46a3601a48185e0

As discussed, I have removed the legacy config for AWS and Firebase deployments available in MC bundles project and pointed the users to the MC official documentation for deployment examples with some hints.